next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: false,
+  images: { unoptimized: true },
```

Comment:
You can also remove this, we are not dealing with images right now. (In future maybe)